### ISSUE TITLE
Goto cannot in

### ISSUE THREAD

**Issue by coffee95** [antest1/GotoBrowser#47]
![Screenshot_20210205-211813_GotoBrowser](https://user-images.githubusercontent.com/78655328/107123904-5a79f280-68db-11eb-9207-5730db978d85.jpg)

After 5 February 2021 maint, my goto cannot go om amd keep show tgis issue

**Comment by laplamgor** [comment]
Are you using any proxy?

Does reinstalling goto help?

**Comment by coffee95** [comment]
No, i dint use any proxy, i try ro reinstall but still same error.

**Comment by XxJimmy122xX** [comment]
You are getting cat bomb No.100. This error code mean "Time unmatched or token expire".
Try clearing cache and play around with the time setting in Android.

**Comment by coffee95** [comment]
Mqy i know what did you mean by 'play around with the time setting in
Android?'

**Comment by XxJimmy122xX** [comment]
Time and time zone settings. Try set it the same as Japan might help.

**Comment by coffee95** [comment]
Ok later go try, thx Sent from Mail for Windows 10 From: XxJimmy122xXSent: Tuesday, February 9, 2021 11:17 PMTo: antest1/GotoBrowserCc: coffee95; AuthorSubject: Re: [antest1/GotoBrowser] Goto cannot in (#47) Time and time zone settings. Try set it the same as Japan might help.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by coffee95** [comment]
I try to set time and clear cache but still no use, re-install again still no use, same error

**Comment by antest1** [comment]
you may try to logout and re-login? (the last button in the left size above X)

**Comment by coffee95** [comment]
also no use
